src/diffusers/models/normalization.py

```diff
@@ -526,3 +526,11 @@ def forward(self, x):
         gx = torch.norm(x, p=2, dim=(1, 2), keepdim=True)
         nx = gx / (gx.mean(dim=-1, keepdim=True) + 1e-6)
         return self.gamma * (x * nx) + self.beta + x
+
+
+class RMSNorm2d(nn.LayerNorm):
```

Comment:
Can we leverage https://github.com/huggingface/diffusers/blob/9a92b8177cb3f8bf4b095fff55da3b45a3607960/src/diffusers/models/normalization.py#L486 here?

Reply:
I think they are not that compatible. Since RMSNorm deals with tensors that have channels as the last dimension. However, RMSNorm2d deals with tensors of format NCHW.

Reply:
Sure.